pkgs/development/python-modules/range-typed-integers/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytestCheckHook }:
```

Easier to review and better readable diffs in the future.

```suggestion
{ lib
, buildPythonPackage
, fetchFromGitHub
, pytestCheckHook
}:
```

---

Do you recommend me doing this for all of the other packages I added?

---

Also, do you know if keeping , in the line of the expression is important? I have some difficulty reading , at the beginning of a line.

---

*Well, if that’s what nixpkgs-fmt use, I’ll use it. I actually look better without the red background for change*